buildSrc/src/main/java/Dependencies.kt

```diff
@@ -44,7 +44,7 @@ object Versions {
     const val mockitoCore = "3.2.4"
     const val mockitoKotlin = "2.2.0"
     const val junit = "4.12"
-    const val adxCore = "1.1.0"
+    const val adxCore = "1.0.0"
```

Comment:
this change should not be needed I guess

Reply:
it didn't work for me before that

Reply:
That is weird! Why would it require to downgrade the lib. It works fine for all of us. Can you send the error you are getting

Reply:
Could not find core.aar (androidx.test:core:1.1.0).
Searched in the following locations:
    https://dl.google.com/dl/android/maven2/androidx/test/core/1.1.0/core-1.1.0.aar

Reply:
this link works though

Reply:
try invalidating the gradle cache and restart ;p